### PR TITLE
Use cmdutil.PrintSuccess() to display bulk output

### PR DESCRIPTION
**Pull request by smarterclayton**
Also remove some extra complexity in new-app output.

@csrwng @bparees

### CODE REVIEW

File: pkg/cmd/cli/cmd/newapp.go [hunk 1]
```diff
@@ -191,31 +192,27 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
...
+				fmt.Fprintf(c.Out(), "Build scheduled for %q - use the build-logs command to track its progress.\n", t.Name)
+			}
 		case *imageapi.ImageStream:
 			if len(t.Status.DockerImageRepository) == 0 {
 				if hasMissingRepo {
 					continue
 				}
 				hasMissingRepo = true
-				fmt.Fprintf(c.Out(), "WARNING: We created an image stream %q, but it does not look like a Docker registry has been integrated with the server. Automatic builds and deployments depend on that integration to detect new images and will not function properly.\n", t.Name)
+				fmt.Fprintf(c.Out(), "WARNING: No Docker registry has been configured with the server. Automatic builds and deployments may not function.\n", t.Name)
```
Comment (kargakis) [inline pkg/cmd/cli/cmd/newapp.go]: Suggest `oadm registry` ?

Comment (csrwng) [inline pkg/cmd/cli/cmd/newapp.go]: The user of this command may not be the administrator. Might cause more confusion than good.

**Comment by mfojtik** [comment]
LGTM

File: pkg/cmd/cli/cmd/newapp.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/newapp.go]: Yeah, we probably just want to warn - they'll know how to ask their admin.

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/newapp.go]: [merge]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**